.travis.yml

```diff
@@ -1,10 +1,20 @@
 dist: xenial
 language: python
 python:
-  - '3.5'
-  - '3.6'
-  - '3.7'
+- '3.5'
```

Comment:
This change was made by the travis gem automatically, when encrypting the API token.